lib/internal/Magento/Framework/Mview/Test/Unit/View/ChangelogTest.php

```diff
@@ -6,6 +6,11 @@
 
 namespace Magento\Framework\Mview\Test\Unit\View;
 
+/**
+ * Class ChangelogTest
+ *
+ * @package Magento\Framework\Mview\Test\Unit\View
```

Comment:
I think this information is redundant.

Better to add comment about general purposes of this class and leave with `@see` reference to class which it cover.